### PR TITLE
Zhijianli88/log tracking

### PR DESCRIPTION
**Pull request by zhijianli88**

### CODE REVIEW

File: keps/sig-instrumentation/1668-log-tracking/README.md [hunk 1]
```diff
@@ -188,38 +172,40 @@ Implement a new inittrace handler to propagate the initialtraceid.
...
+    ctx = httptrace.SpanContextFromAnnotations(objB.GetAnnotations())
     err = r.KubeClient.CoreV1().Create(ctx, objA...)
 ```
 We do propagation across objects without adding traces to that components.
-
-When controllers create/update/delete an object A based on another B, we propagate context from B to A. E.g.:
-```
-    ctx = traceutil.WithObject(ctx, objB)
-    err = r.KubeClient.CoreV1().Create(ctx, objA...)
+**3. Makeup a new outgoing [request](#design-of-id-progagation-in-client-go)**
```
Comment (KobayashiD27) [inline keps/sig-instrumentation/1668-log-tracking/README.md]: Looks like it needs a line break.

File: keps/sig-instrumentation/1668-log-tracking/README.md [hunk 2]
```diff
@@ -188,38 +172,40 @@ Implement a new inittrace handler to propagate the initialtraceid.
...
+
+### Design of ID propagation in [client-go](https://github.com/kubernetes/client-go)
+client-go  helps to inject [TraceContext](https://pkg.go.dev/go.opentelemetry.io/otel/propagators#example-TraceContext) and [Baggage](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/baggage/api.md) to the outgoing http request header, something changes are like below:
+```diff
+@@ -868,6 +871,7 @@ func (r *Request) request(ctx context.Context, fn func(*http.Request, *http.Resp
```
Comment (KobayashiD27) [inline keps/sig-instrumentation/1668-log-tracking/README.md:185]: Are the line numbers needed?

File: keps/sig-instrumentation/1668-log-tracking/README.md [hunk 3]
```diff
@@ -188,38 +172,40 @@ Implement a new inittrace handler to propagate the initialtraceid.
...
 
 ### Design of Mutating webhook(Out of tree)
 
-**1. Extract SpanContext and initialtraceid from request's header**
+**1.Extract SpanContext and initialtraceid from request's header **
 
-**2. Update SpanContext and initialtraceid to object**
+**2.Update SpanContext and initialtraceid to object**
 
 - For traceid and spanid
```
Comment (KobayashiD27) [inline keps/sig-instrumentation/1668-log-tracking/README.md]: Could you add the trace-flag?
Fix to:
`For traceid, spanid, and trace flags (sampled/not sampled)`

**Review by KobayashiD27 — COMMENTED** [review]
I found some fix points.

File: keps/sig-instrumentation/1668-log-tracking/README.md (hunk 2 above)
Comment (zhijianli88) [reply keps/sig-instrumentation/1668-log-tracking/README.md:185]: Yes, there are part of `git diff`

File: keps/sig-instrumentation/1668-log-tracking/README.md (hunk 3 above)
Comment (zhijianli88) [reply keps/sig-instrumentation/1668-log-tracking/README.md]: do you mind to use traceflags
```
type SpanContext struct {
	TraceID    ID
	SpanID     SpanID
	TraceFlags byte
}
```

File: keps/sig-instrumentation/1668-log-tracking/README.md [hunk 4]
```diff
@@ -381,3 +368,5 @@ _This section must be completed when targeting beta graduation to a release._
 
 * 2020-09-01: KEP proposed
 * 2020-09-28: PRR questionnaire updated
+* [Mutating admission webhook which injects trace context for demo](https://github.com/Hellcatlk/mutating-trace-admission-controller/tree/trace-ot)
+* [Instrumentation of Kubernetes components for demo](thttps://github.com/Hellcatlk/kubernetes/tree/trace-ot)
```
Comment (KobayashiD27) [inline keps/sig-instrumentation/1668-log-tracking/README.md]: I found link error, please fix it.

**Pull request merged**